src/backend/tests/app/test_flask_app.py

```diff
@@ -0,0 +1,35 @@
+import unittest
+import expungeservice
+from flask import g
+
+
+class TestFlaskApp(unittest.TestCase):
+
+    def setUp(self):
+        self.app = expungeservice.app.create_app('development')
+        self.test_client = self.app.test_client()
+
+    def tearDown(self):
+        pass
+
+    def test_get_database_connection(self):
+
+        with self.app.app_context():
+
+            expungeservice.request.before()
+
+            assert g.database
+            
+            query = 'SELECT * FROM users;'
+            g.database.cursor.execute(query, ())
+            rows = g.database.cursor.fetchall()
+            assert rows or rows == []
+
+            expungeservice.request.teardown(None)
+            assert g.database.cursor.closed
+            
+    def test_hello_request(self):
```

Comment:
This is a good test to have in the endpoint directory. I don't think it belongs here. Plus it will be breaking when remove the hello endpoint in the future. Can we get it moved to the `tests/endpoints/hello`?

Reply:
I did hesitate to put this in, but I wanted a test that verifies the connection succeeds during an actual endpoint call... um but this test doesn't explicitly do that (only verifies that an error isn't thrown during the attempt). Is it is indeed a useless test since it only tests a toy feature. The other tests on endpoint code definitely belong elsewhere. I will remove this.

Reply:
ok